services/director-v2/tests/mocks/fake_scheduler_data.json

```diff
@@ -0,0 +1,55 @@
+{
+    "paths_mapping": {
+        "inputs_path": "/home/jovyan/work/inputs",
+        "outputs_path": "/home/jovyan/work/outputs",
+        "state_paths": [
+            "/home/jovyan/work/workspace"
+        ],
+        "state_exclude": null
+    },
+    "compose_spec": "null",
+    "container_http_entry": null,
+    "restart_policy": "no-restart",
+    "key": "simcore/services/dynamic/jupyter-math",
+    "version": "2.0.4",
+    "user_id": 1,
+    "project_id": "c440556a-b4d9-11ec-9a17-02420a000016",
+    "node_uuid": "3e68d1f6-be3e-414e-a468-4a2bf415f756",
+    "service_name": "dy-sidecar_3e68d1f6-be3e-414e-a468-4a2bf415f756",
+    "dynamic_sidecar": {
+        "status": {
+            "current": "ok",
+            "info": ""
+        },
+        "hostname": "dy-sidecar_3e68d1f6-be3e-414e-a468-4a2bf415f756",
```

Comment:
simple question: what happens if the node where the sidecar was goes down? is the service moving to another one? will it fail?

Reply:
Some notes:
- When a dynamic-sidecar is started is paired together with a proxy which is scheduled to the same node where the dynamic-sidecar is running.
- Since moving the proxy from Traefik to Caddy, it is no longer a requirement to be on the same node.
- If the service gets removed, all its spawned containers are are also removed. So cleanup is OK.

Now if the node on which the dynamic-sidecar is running goes down the director-v2 will no longer see the service can no longer contact it. Will mark is as unhealthy and try to remove it from tracking. Will have to check this.